lib/menu.rb

```diff
@@ -0,0 +1,69 @@
+require 'json'
+
+class Menu
+
+     attr_accessor :customer_bill, :customer_order, :taxes, :sum, :amount
+     attr_reader :menu
+
+  def initialize
+       file = open("hipstercoffee.json")
+       json = file.read
+       @menu = JSON.parse(json)
+       @customer_order = []
+       @customer_bill = []
+       @total = 0
+       @taxes = 0
+       @quantity = 1
+
+  end
+
+  def order(booking, quantity)
+       @menu[0]["prices"][0].select do |key,value|
+          if booking == key
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)